src/interaction/interactionmode.cpp

```diff
@@ -262,74 +263,77 @@ OrbitalInteractionMode::MouseStates::MouseStates(double sensitivity, double velo
 {}
 
 void OrbitalInteractionMode::MouseStates::updateMouseStatesFromInput(const InputState& inputState, double deltaTime) {
-    glm::dvec2 mousePosition = inputState.getMousePosition();
+    ghoul::any t = property("Global Properties.Touch.TouchInteraction.TouchEvents")->get();
```

Comment:
I'm not sure, if the module doesn't exist `ghoul::any t` will return false and should thus work fine without included module as well. I can encapsulate the start and end of the if-statement with module-checks for safety.